### PR TITLE
added Player enum

### PR DESCRIPTION
**Pull request by rachel613g**

### CODE REVIEW

File: src/main/java/pong/Player.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package pong;
+
+public enum Player {
+    HUMAN, COMPUTER
```
Comment (lliebman) [inline src/main/java/pong/Player.java]: would USER be better than HUMAN?

Comment (rachel613g) [reply src/main/java/pong/Player.java]: I changed it to this in a more recent commit.

**Comment by rachel613g** [comment]
If you deem this ready it can be merged now. I thought I wanted to change something but I was mistaken

File: src/main/java/pong/Player.java (hunk 1 above)
Comment (lliebman) [reply src/main/java/pong/Player.java]: Not sure what you mean. You updated it to user in a more recent commit?

Comment (rachel613g) [reply src/main/java/pong/Player.java]: Sorry, you are right it is fixed now in commit [e3b7c13](https://github.com/lliebman/Pong/pull/4/commits/e3b7c130321d8c91261a3abb6ff1d5e20209f16d)

**Review by lliebman — APPROVED** [review]

**Pull request merged**